Samples/Tools/CMakeGDKExample/CMakeLists.txt

```diff
@@ -39,8 +39,8 @@ set(GAME_ASSETS
 
 source_group("Assets" FILES ${GAME_ASSETS})
 
-# MicrosoftGame.config
-set_source_files_properties(MicrosoftGame.Config PROPERTIES VS_TOOL_OVERRIDE "MGCCompile")
+# MicrosoftGameConfig.mgc
+set_source_files_properties(MicrosoftGameConfig.mgc PROPERTIES VS_TOOL_OVERRIDE "MGCCompile")
```

Comment:
I see ``MicrosoftGame.config`` file is deleteed, but where is the rename ``MicrosoftGameConfig.mgc``?

Reply:
It doesn't look like the mgc version was distributed for either CMakeGDKExample or CMakeXboxConsoleApp.

Reply:
There isn't one for CMakeXboxConsoleApp, but I'll submit a fix for CMakeGDKExample.